app/src/test/java/ru/vssemikoz/newsfeed/newsfeed/NewsFeedPresenterTest.java

```diff
@@ -84,53 +90,62 @@ private String generateRandomString(int stringSize) {
     }
 
     @Test
-    public void verifyUpdateActualNews_ShowProgressBarIsCalled() {
+    public void verifyUpdateActualNews_UpdateStorageUseCaseIsCalled() {
+        when(updateStorageUseCase.run(any())).thenReturn(Single.just(exampleNewsList));
+        when(getFilteredNewsUseCase.run(any())).thenReturn(Single.just(exampleNewsList));
         presenter.updateActualNews();
-        verify(view).showProgressBar();
+        verify(updateStorageUseCase).run(any());
```

Comment:
What cases are you going to cover? I see these ones:
1. All interaction started in subscribe: 
1.1. you should verify that your code in `initStartValues` gets called; 
1.2. you should verify that after `subscribe` on successful network operation the emit will be processed. It'd be nice to wrap `onNext` callback code into the single method.
1.3. you should verify that after `subscribe` on failed network/storage operations the failure callback will be processed. It'd be nice to wrap `onError` callback code into the single method.
2. `openNewsDetails` calls navigator.
3. `shareNewsItem` calls navigator.
4. `changeNewsFavoriteState` initiates list rendering. See related comment for `presenter.changeNewsFavoriteState`.
5. `invertFavoriteState` initiates list rendering.

Others are subject to be removed. Please, review your tests and remove unnecessary ones.